src/routes/area/index.svelte

```diff
@@ -6,24 +6,58 @@
   import UseCensusData from "../../ui/UseCensusData.svelte";
   import Feedback from "../../ui/Feedback.svelte";
   import DataHeader from "../../ui/DataHeader.svelte";
+  import Header from "../../ui/Header.svelte";
+  import ExploreByAreaComponent from "../../ui/ExploreByAreaComponent.svelte";
   import ONSShareItem from "../../ui/ons/partials/ONSShareItem.svelte";
   import ONSFacebookIcon from "../../ui/ons/svg/ONSFacebookIcon.svelte";
   import ONSTwitterIcon from "../../ui/ons/svg/ONSTwitterIcon.svelte";
   import ONSLinkedinIcon from "../../ui/ons/svg/ONSLinkedinIcon.svelte";
   import ONSEmailIcon from "../../ui/ons/svg/ONSEmailIcon.svelte";
   import { page } from "$app/stores";
-  import { getLadName, updateSelectedGeography, updateHoveredGeography } from "../../model/geography/geography";
+  import {
+    getLadName,
+    updateSelectedGeography,
+    updateHoveredGeography,
+    reverseLadLookup,
+  } from "../../model/geography/geography";
   import { appIsInitialised } from "../../model/appstate";
   import { areaSelectedTopicSuggestions } from "../../config";
   import config from "../../config";
   import TileSet from "../../ui/map/TileSet.svelte";
   import InteractiveLayer from "../../ui/map/InteractiveLayer.svelte";
   import BoundaryLayer from "../../ui/map/BoundaryLayer.svelte";
+  import { goto } from "$app/navigation";
 
-  const locationId = $page.query.get("location");
-  let locationName;
+  let locationName, locationId;
   let topicSuggestions;
 
+  let autosuggestData = "https://raw.githubusercontent.com/ONSdigital/census-atlas/master/src/data/ladList.json";
```

Comment:
Isn't this block of code duplicated in the [categorySlug].svelte?
Is there a way it could be made into common/shared code?

Reply:
Yes it showing up on multiple routes and I believe we could move it in src/config.js file ?

Reply:
I have moved the duplicated code into a wrapper component `<HeaderWrapper />` which is imported and used on both routes/area/**index.svelte** and routes/[topicSlug]/[tableSlug]/**[categorySlug].svelte** pages.